pages/common/rustup-toolchain.md

```diff
@@ -19,3 +19,7 @@
 - Create a custom toolchain by symlinking to a directory:
 
 `rustup link {{custom_toolchain_name}} {{path/to/directory}}`
+
+- Display help:
+
+`rustup help toolchain`
```

Comment:
Is this example really needed, given we have this in the description?

> See `rustup help toolchain` for more information about toolchains.

```suggestion
```

Reply:
IMO we can remove it.